code/src/index.css

```diff
@@ -1,13 +1,246 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: 'Fanwood Text', serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.main-container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-image: url(assets/start.jpg);
+	height: 100vh;
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position-y: 1%;
+	background-position-x: 50%;
+	position: relative;
+}
+
+.start-container {
+	background-color: #d8bf978a;
+	border-radius: 25px;
+	display: flex;
+	flex-direction: column;
+	padding: 11px;
+	width: 85%;
+	color: #292b1d;
+	align-items: center;
+}
+
+.start-container h1 {
+	font-family: 'Ingrid Darling';
+	color: #292b1d;
+	font-size: 50px;
+	margin: 0;
+}
+
+.start-container p {
+	font-family: 'Fanwood Text', serif;
```

Comment:
You already set this font-family in the body, so you don't have to set it here again. :)